st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -88,7 +91,19 @@ def post(self, payload):
                                              notify=notify,
                                              context=context)
 
-        return str(execution.id)
+        result = {
+            'execution': execution,
+            'actionalias': ActionAliasAPI.from_model(action_alias_db)
+        }
+
+        if action_alias_db.ack and 'format' in action_alias_db.ack:
+            jinja = jinja2.Environment(trim_blocks=True, lstrip_blocks=True)
```

Comment:
These two lines, as well as the "in" comparison, is mostly a matter of convenience and personal preference. If you're ok with transferring them into util (trimming just makes sense to get rid of excessive whitespace, and "in" is just useful), I can use the method. In fact, using util is probably better anyway, for the sake of consistency.